isso/views/comments.py

```diff
@@ -351,6 +351,8 @@ def _remote_addr(self, request):
     """
     @api {get} /id/:id view
     @apiGroup Comment
+    @apiDescription
+        View an existing comment, for the purpose of editing. Editing a comment is only possible for a short period of time after it was created and only if the requestor has a valid cookie for it. See the [isso server documentation](https://posativ.org/isso/docs/configuration/server) for details.
```

Comment:
Maybe change the `apiExample` to
`curl 'https://comments.example.com/id/4' --cokie '4=<cookie>'`

Reply:
Used `-b cookie.txt` to keep it in line with the other `@apiExample` annotations